pom.xml

```diff
@@ -0,0 +1,355 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>science.aist.sushiya</groupId>
+    <artifactId>sushiya</artifactId>
+    <packaging>pom</packaging>
+    <version>1.0.0-SNAPSHOT</version>
+    <scm>
+        <developerConnection>scm:git:git@github.com:FHOOEAIST/sushiya.git</developerConnection>
+        <tag>HEAD</tag>
+    </scm>
+    <distributionManagement>
+        <repository>
+            <id>maven-releases</id>
+            <url>https://aist.fh-hagenberg.at/nexus/repository/maven-releases/</url>
+        </repository>
+        <snapshotRepository>
+            <id>maven-snapshots</id>
+            <url>https://aist.fh-hagenberg.at/nexus/repository/maven-snapshots/</url>
+        </snapshotRepository>
+        <site>
+            <id>javadoc</id>
+            <url>dav:https://aist.fh-hagenberg.at/nexus/repository/javadoc/sushiya/${project.version}</url>
+        </site>
+    </distributionManagement>  <!-- variables -->
+    <properties>    <!-- aist dependencies -->
+        <seshat.version>1.1</seshat.version>
+        <jfuck.version>1.1</jfuck.version>    <!-- further dependencies -->
```

Comment:
see comment above to jfuck -> https://github.com/FHOOEAIST/jack

Reply:
@pointan looks like we need to exchange this in the Archetype with the public maven dependency

Reply:
Jup, seems like this hasn't been touched for a while. As jfuck has been renamed to jack, one year ago.